extensions/validation/src/test/java/org/mule/extension/validation/ValidationTestCase.java

```diff
@@ -26,7 +26,6 @@
 
     static final String VALID_URL = "http://localhost:8080";
     static final String INVALID_URL = "here";
-
```

Comment:
I don't see why we should have lines between constants, a formatter tool will never understand that.